### DISCUSSION TITLE
How can I make the files stay on the local server and download from there?

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3858] [Q&A]
Hi, I'm doing a harvest and at the moment it seems to work fine, but I have a problem and that is that when I go to the dataset site, the files contain the url, from where the harvest was made, and not from the local machine, It doesn't stay with my domain of "dataforeverybody.com" but it stays with the IP of the place of the harvest attached image, I'll be attentive, thank you very much.

I'm checking in the drush options but I can't find anything related, in the dkan part

**Comment by tridoxx** [comment]
using the next comand it will harvest inside the machine. remember change the uri in this case  "http://web-com/data.json" 

drush dkan:harvest:register '{ "identifier": "webharvest", "extract": { "type": "\\Harvest\\ETL\\Extract\\DataJson", "uri": "http://web-com/data.json" }, "transforms": ["\\Drupal\\harvest\\Transform\\ResourceImporter"], "load": { "type": "\\Drupal\\harvest\\Load\\Dataset" } }'

**Comment by janette** [comment]
@tridoxx there is a setting to use the local copy of the resource for the download url:

![Resources___DKAN](https://github.com/GetDKAN/dkan/assets/314172/18829ea8-d874-4adc-8d22-f3fc8bb77851)
